doc: add the content of hugepages/hugepagesz in generic kernel parame… (#6155)
projectacrn/acrn-hypervisor: doc/user-guides/kernel-parameters.rst

         intel_iommu=off

   * - ``hugepages``
       ``hugepagesz``
     - Service VM,User VM
     - ``hugepages``:
         HugeTLB pages to allocate at boot.

       ``hugepagesz``:

>          The size of the HugeTLB pages. On x86-64 and powerpc,

Reviewer: ```suggestion
         The size of the HugeTLB pages. On x86-64 and PowerPC,
```

Author: Updated.

         this option can be specified multiple times interleaved
         with ``hugepages`` to reserve huge pages of different sizes.

>          Valid pages sizes on x86-64 are 2M (when the CPU supports "pse")

Reviewer: ```suggestion
         Valid page sizes on x86-64 are 2M (when the CPU supports Page Size Extension (PSE))
```

Author: Updated.

>          and 1G (when the CPU supports the "pdpe1gb" cpuinfo flag).

Reviewer: ```suggestion
         and 1G (when the CPU supports the ``pdpe1gb`` cpuinfo flag).
```

Author: Updated.

         intel_iommu=off

   * - ``hugepages``
       ``hugepagesz``
     - Service VM,User VM
     - ``hugepages``:
         HugeTLB pages to allocate at boot.

       ``hugepagesz``:
         The size of the HugeTLB pages. On x86-64 and PowerPC,
         this option can be specified multiple times interleaved
         with ``hugepages`` to reserve huge pages of different sizes.
         Valid page sizes on x86-64 are 2M (when the CPU supports Page Size Extension (PSE))
         and 1G (when the CPU supports the ``pdpe1gb`` cpuinfo flag).
     - ::

         hugepages=10
         hugepagesz=1G

.. note:: The ``hugepages`` and ``hugepagesz`` parameters are automatically
   taken care of by ACRN config tool. In case user have customized hugepage

>    settings to satisfy their particular workloads in SOS VM, the ``hugepages``

Reviewer: ```suggestion
   settings to satisfy their particular workloads in Service VM, the ``hugepages``
```

Author: Updated it.
